lib/model/query/entities.js

```diff
@@ -96,9 +96,13 @@ const processSubmissionEvent = (event) => (container) =>
     .then((entity) => {
       if (entity != null) {
         return container.Audits.log({ id: event.actorId }, 'entity.create', { acteeId: entity.aux.dataset.acteeId },
-          { entity: { uuid: entity.uuid, label: entity.def.label, dataset: entity.aux.dataset.name },
+          {
+            entityId: entity.id,
```

Comment:
Do we need to backfill `entityId`? Or are we OK with the entity activity feed being empty for entities created before .3?

Reply:
We've decided not to backfill `entityId`. It feels best to avoid changing the audit log when possible so that it can serve as a historical record.

Similar to my comment below, would you mind leaving a comment in the code that mentions that this property was added in v2023.3?